README.md

```diff
@@ -17,15 +17,98 @@ const userConnections = {
   type: userConnection,
   args: connectionArgs,
   resolve: ((_, args) => {
-    promisifiedDocumentClient.scan({
+    return promisifiedDocumentClient.scan({
       TableName: 'users',
       ...paginationToParams(args)
     })
-    .then(data => dataToConnection(data));
+    .then(data => {
+      return dataToConnection(data)
+    });
   })
 };
 ```
 
+### And a more complete example using native ES2015 Node.js features
```

Comment:
It would also be great to have a comment on top to explain what this example does for people who are just starting to play with GraphQL and this library specifically.